lib/vsc/utils/fancylogger.py

```diff
@@ -450,18 +450,28 @@ def logToFile(filename, enable=True, filehandler=None, name=None, max_bytes=MAX_
 
     if you want to disable logging to file, pass the earlier obtained filehandler
     """
-    handleropts = {'filename': filename,
-                   'mode': 'a',
-                   'maxBytes': max_bytes,
-                   'backupCount': backup_count,
-                   }
-    return _logToSomething(logging.handlers.RotatingFileHandler,
-                           handleropts,
-                           loggeroption='logtofile_%s' % filename,
-                           name=name,
-                           enable=enable,
-                           handler=filehandler,
-                           )
+    handleropts = {
+        'filename': filename,
+        'mode': 'a',
+        'maxBytes': max_bytes,
+        'backupCount': backup_count,
+    }
+    # logging to a file is going to create the file later on, so let's try to be helpful and create the path if needed
+    directory = os.path.dirname(filename)
+    if not os.path.exists(directory):
+        try:
+            os.makedirs(directory)
+        except Exception as ex:
+            raise sys.exc_info()[0], "Cannot create logdirectory %s: %s" % (directory, ex), sys.exc_info()[2]
```

Comment:
I'm not generating the traceback here, just preserving the original one, the fact that this is seen by the user is because there's no try-except around my call to logToFile, but as you can see, it will raise an error and print this traceback wether or not this code is in place, so this code is an improvement, and should be merged.